x-pack/heartbeat/Jenkinsfile.yml

```diff
@@ -21,11 +21,14 @@ stages:
           make -C heartbeat update;
           make check-no-changes;
         stage: lint
-    build:
-        mage: "mage build test"
+    unitTest:
+        mage: "mage build unitTest"
+        stage: mandatory
+    goIntegTest:
+        mage: "mage goIntegTest"
         stage: mandatory
     macos:
-        mage: "mage build test"
+        mage: "mage build unitTest"
```

Comment:
What's the reason on Macos?

Reply:
I'd say consistency across platforms. I.e. #20104 added this block for auditbeat:

```groovy
macos:
        mage: "mage build unitTest"
```

Reply:
Maybe there is requirement to avoid that. If so, please let me know to revert this part for heartbeat

Reply:
I didn't recall it was `unitTest` in macos by default. https://github.com/elastic/beats/pull/23873 was the one that actually added that behaviour. 
+100 for the consistency, thanks for the clarification